src/main/resources/config/application-inmem.yml

```diff
@@ -0,0 +1,8 @@
+spring:
+  profiles:
+    include: default
+
+security:
+  oauth2:
+    resource:
+      tokenInfoUri: "https://info.services.auth.zalando.com/oauth2/tokeninfo"
```

Comment:
Good catch. Will change.

On Tue, 15 Mar 2016, 09:52 Henning Jacobs, <notifications@github.com> wrote:

> In src/main/resources/config/application-inmem.yml
> <https://github.com/zalando/planb-revocation/pull/61#discussion_r56130906>
> :
>
> > @@ -0,0 +1,8 @@
> > +spring:
> > +  profiles:
> > +    include: default
> > +
> > +security:
> > +  oauth2:
> > +    resource:
> > +      tokenInfoUri: "https://info.services.auth.zalando.com/oauth2/tokeninfo"
>
> Can we please use example URLs here? Company specific URLs should not
> appear in this repo, you might set them via environment variable.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/zalando/planb-revocation/pull/61/files#r56130906
>